datadotworld/client/api.py

```diff
@@ -723,7 +729,7 @@ def sql(self, dataset_key, query, desired_mimetype="application/json", **kwargs)
 
     # Sparql Operations
 
-    def sparql(self, dataset_key, query, desired_mimetype="application/json"):
+    def sparql(self, dataset_key, query, desired_mimetype="application/json", **kwargs):
```

Comment:
I'm seeing double quotes around strings in a few places. We try to use single quotes only.